File: materialm-angular-free-main/main/src/app/Models/CompanyDTO.ts
import { CandlestickData } from "./CandlestickData";
import { News } from "./News";

export class CompanyDTO {
    
    name!: string;
    industry!: string;
    initialPrice!: number;
    volatility!: number;
    drift!: number;
    candlestickData!:CandlestickData[];
    news!: News[];
}

File: Pi5/src/main/java/org/example/pi5/entities/CompanyDTO.java
package org.example.pi5.entities;

import lombok.*;

import java.util.List;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class CompanyDTO {
    private String name;
    private String industry;
    private double initialPrice;
    private double volatility;
    private double drift;
    private List<CandlestickData> candlestickData;
    private List<News> news;
}
